core/src/main/java/jenkins/management/ShutdownLink.java

```diff
@@ -28,38 +28,61 @@
 import hudson.model.ManagementLink;
 import hudson.security.Permission;
 import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
 import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.verb.POST;
+
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 
-/**
- * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
- */
-@Extension(ordinal = Integer.MIN_VALUE) @Symbol("shutDown")
+@Extension(ordinal = Integer.MIN_VALUE)
+@Symbol("prepareShutdown")
 public class ShutdownLink extends ManagementLink {
 
+    private static final Logger LOGGER = Logger.getLogger(ShutdownLink.class.getName());
+
     @Override
     public String getIconFileName() {
         return "system-log-out.png";
     }
 
     public String getDisplayName() {
-        return Jenkins.get().isQuietingDown() ? Messages.ShutdownLink_DisplayName_cancel() : Messages.ShutdownLink_DisplayName_prepare();
+        return Messages.ShutdownLink_DisplayName_prepare();
```

Comment:
I'm unsure about this UI change.

It makes sense that the management of the quiet down reason is placed in the `ManagementLink` form, but it is confusing to see the link "Prepare for shutdown" when we're already in this state.

I wonder whether we could retain the existing behavior, perhaps with a different label instead of the now not-quite-applicable "Cancel Shutdown", like "Update shutdown preparation"?

Reply:
- [ ] Show different text for link to shutdown page